bestie/proto/test_metrics.proto

```diff
@@ -0,0 +1,60 @@
+// Chose proto3 primarily for compatibility with our infrastructure.
+syntax = "proto3";
+
+package bestie.proto;
+
+option go_package = "github.com/enfabrica/enkit/bestie/proto";
+
+// BigQuery table identifier.
+//    project - the Google Cloud billing entity set up for the organization to use
+//    dataset - logical container that holds tables and views; each group may create its own
+//    tablename - name of the table to hold these metrics; the same tablename is used across datasets
+message BigQueryTable {
+    string project = 1;
+    string dataset = 2;
+    string tablename = 3;
+}
+
+// BigQuery metric definition compatible with Prometheus remote read adapter.
+//
+// The 'tags' field is a freeform, JSON-like string with comma-separated key=value pairs.
```

Comment:
should we use something like:

```
repeated KeyValue tags;
```
with:
```
message KeyValue {
  string key = 1;
  string value = 2;
}
```
?

We can then turn it into a single json field for bigquery easily, or do other automated processing more easily in the ingestion pipeline if necessary?

(this also guarantees that the value is syntactically valid, and saves the caller from having to implement any form of escaping, which is error prone. Eg, let's say otherwise this field contains invalid values, how do we return and inform the caller of an error? Hard to do from the BES endpoint, we risk having a silent error cause data to be discarded, hard to debug)

Reply:
Agreed. Made this change in the latest commit.